spec/models/instance/settings_spec.rb

```diff
@@ -1,45 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Instance::Settings, type: :model do
-  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F]
-  ENABLED_COMPONENTS = [:A, :C, :E]
+  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F].freeze
```

Comment:
You have no way to access the constant afterward, would you? You cannot qualify the namespace the constant belongs to.